src/Adapter/Debug/DebugModeConfiguration.php

```diff
@@ -116,22 +166,7 @@ public function updateConfiguration(array $configuration)
                     ];
 
                     break;
-                case DebugMode::DEBUG_MODE_ERROR_NO_WRITE_ACCESS_CUSTOM:
-                    $errors[] = [
-                        'key' => 'Error: Could not write to file. Make sure that the correct permissions are set on the file %s',
-                        'domain' => 'Admin.Advparameters.Notification',
-                        'parameters' => [$this->configDefinesPath],
-                    ];
-
-                    break;
-                case DebugMode::DEBUG_MODE_ERROR_NO_READ_ACCESS:
-                    $errors[] = [
-                        'key' => 'Error: Could not write to file. Make sure that the correct permissions are set on the file %s',
```

Comment:
I see. This is outside of the PR scope but it's cleaning.

Don't hesitate to explain this, I've been searching for 5 minutes why you were doing this 😄  because I saw not link to PR goal.